charts/generic-prometheus-alerts/ci/tests/application-alerts-tests.yaml

```diff
@@ -3,7 +3,7 @@ rule_files:
   - ../test-application-app.yaml
   - ../test-business-hours-app.yaml
 
-evaluation_interval: 5m
+evaluation_interval: 1m
```

Comment:
needed so that our count over last 5 minutes works - otherwise we just evaluate at 0 and again at 5m